### ISSUE TITLE
FNX-3254 ⁃ [Bug] In Settings -> Accessibility -> Font Size, the values are incorrect for screen reader users

### ISSUE THREAD

**Issue by asadotzler** [mozilla-mobile/fenix#10049]
## Summary
Fenix allows users to set a custom font size in the Accessibility settings. After disabling Automatic Font Sizing, the user is presented with a Font Size slider control that allows them to select a font size that is somewhere between 50% and 200% with 100% as the default value. Visually, these values are displayed at the right end of the slider control. These values are accurate as the displayed sample text attests.

For the screen reader user, however, the default value is spoken as "thirty-three percent" and when the slider is at its lowest value, the screen reader user hears "zero percent" and at it's highest value they will hear "one hundred percent" These values are not accurate as the displayed sample text attests. 

The screen reader user should hear accurate values for the text zoom level.

## Steps to reproduce
Enable TalkBack and open Fenix
Navigate to the Accessibility Settings and turn off Automatic Font Sizing
Focus the Font Size slider control and listen for the initial value.
Change the value with the volume rocker or the edit seek control level context menu

### Expected behavior
TalkBack should announce 100% as the initial value and 50% as the minimum value and 200% as the maximum value. 

### Actual behavior
TalkBack announces as if the scale was 0% to 100% instead of 50% to 200%

### Device information
* Android device: Galaxy S10e
* Fenix version: 4/17

**Comment by asadotzler** [comment]
Also, the edit seek control level context menu, reached with a swipe up and right, only allows values of zero to one hundred when what the user actually wants is fifty to two hundred.

**Comment by mcarare** [comment]
> Also, the edit seek control level context menu, reached with a swipe up and right, only allows values of zero to one hundred when what the user actually wants is fifty to two hundred.

Wouldn't this feature be more accessible if there were also two buttons to increase or decrease the text scale?

**Comment by mcarare** [comment]
@asadotzler While the issue in the first comment can be fixed, I'm afraid that that dialog has those values hardcoded in Talkback API : `    <string name="value_seek_bar_dialog_instructions">Enter a number from 0 to 100.</string>
.` The transposal of the value inserted is also handled internally.
Maybe we could avoid the use of that menu dialog by providing controls like I suggested above?

**Comment by abodea** [comment]
Verified as fixed on the latest **Nightly 6/2**3 with **Google Pixel 4 XL (10).**
